test/e2e/tests/connections/edit-account-flow.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { withFixtures, WINDOW_TITLES } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import { DEFAULT_FIXTURE_ACCOUNT, DAPP_HOST_ADDRESS } from '../../constants';
+import AccountListPage from '../../page-objects/pages/account-list-page';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+import Homepage from '../../page-objects/pages/home/homepage';
+import PermissionListPage from '../../page-objects/pages/permission/permission-list-page';
+import SitePermissionPage from '../../page-objects/pages/permission/site-permission-page';
+import TestDapp from '../../page-objects/pages/test-dapp';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+
+const accountLabel2 = '2nd custom name';
+const accountLabel3 = '3rd custom name';
+describe('Edit Accounts Flow', function () {
```

Comment:
nit: for the same reason as above

```suggestion
describe('Edit Accounts Permissions', function () {
```

Reply:
Done, thanks !